Dockerfile

```diff
@@ -14,5 +14,9 @@ ENV BIGCHAINDB_CONFIG_PATH /data/.bigchaindb
 ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
 ENV BIGCHAINDB_WSSERVER_HOST 0.0.0.0
 ENV BIGCHAINDB_WSSERVER_SCHEME ws
+
+ENV BIGCHAINDB_WSSERVER_ADVERTISED_SCHEME ws
+ENV BIGCHAINDB_WSSERVER_ADVERTISED_HOST 0.0.0.0
```

Comment:
This was as it was in the first version of Dockerfile that I dealt with; and it has remained that way from then.
And extending that, I have added the `WSSERVER_ADVERTISED_*` settings accordingly.
Maybe you can file a separate issue for it and we can handle the Dockerfile separately?